File: 0001-two-sum/0001-two-sum.cpp

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans;
        for(int i = 0; i < nums.size(); i++) {
            auto it = find(nums.begin(), nums.end(), target - nums[i]) - nums.begin();
            if(find(nums.begin(), nums.end(), target - nums[i]) != nums.end() && it != i) {
                ans.push_back(i);
                ans.push_back(it);
                break;
            }
            else
                continue;
        }
        return ans;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        sol = []
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                if nums[j] + nums[i] == target:
                    sol.append(i)
                    sol.append(j)
                    return sol
        return []
           